test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
I don't blame you. It's quite evil of Git/GitHub to suggest the wrong context here …